app/src/interfaces/useBioHubApi-interfaces.ts

```diff
@@ -102,3 +102,23 @@ export interface IMedia {
   file_name: string;
   encoded_file: string;
 }
+
+/**
+ * Upload project artifacts request object.
+ * @interface IUploadProjectArtifactRequest
+ */
+export interface IUploadProjectArtifactsRequest {
+  artifacts: {
+    file_name: string;
+    encoded_file: string;
+  }[];
+}
```

Comment:
It looks like this isn't used anywhere, may as well remove.

Reply:
done